builtin/logical/pki/path_root.go

```diff
@@ -408,7 +408,7 @@ func (b *backend) pathIssuerSignIntermediate(ctx context.Context, req *logical.R
 		// If the operator hasn't configured any of the URLs prior to
 		// generating this issuer, we should add a warning to the response,
 		// informing them they might want to do so and re-generate the issuer.
-		resp.AddWarning("This mount hasn't configured any authority access information fields; this may make it harder for systems to find missing certificates in the chain or to validate revocation status of certificates. Consider updating /config/urls with this information.")
+		resp.AddWarning("This mount hasn't configured any authority access information (AIA) fields; this may make it harder for systems to find missing certificates in the chain or to validate revocation status of certificates. Consider updating /config/urls or the newly generated issuer with this information.")
```

Comment:
Suggesting updating the newly generated issuer contradicts the comment above, suggesting that the issuer should be re-generated.  Is it possible to split here, depending on whether this is a root or intermediate CA?

Reply:
No, the comment is wrong. :P Roots generally don't need AIAs, since they are terminal and would merely be self-referential (the issuer storage entry _holds_ AIAs it places on leaves, but its own cert encoding has AIAs it acquired from its parent; in the case of the root, some are self-referential and some are missing, usually its up to operator preference. Empty seems cleaner tbh).